core/src/test/java/org/apache/calcite/test/MaterializedViewRelOptRulesTest.java

```diff
@@ -338,6 +341,104 @@ protected final MaterializedViewFixture sql(String materialize,
         .ok();
   }
 
+  @Test void testAggregateMaterializationAggregateFuncsRange1() {
```

Comment:
Please find better names for tests, if you have multiple tests, they are covering something specific each, so why don't put that in the test name? This is true for all tests1...n in the PR

Reply:
changed test names to be more descriptive